communication/include/communication/StateControl.hpp

```diff
@@ -29,9 +30,15 @@ class StateControl {
   }
 
   void restartComm(uint32_t id) { comm_restart_cb_registry_.invokeAll(id); }
+  void setGetPeerPubKeyMethod(std::function<std::string(uint32_t)> m) { getPeerPubKey_ = std::move(m); }
+  std::string getPeerPubKey(uint32_t id) {
+    if (getPeerPubKey_) return getPeerPubKey_(id);
+    return std::string();
+  }
 
  private:
   std::mutex lock_comm_;
   concord::util::CallbackRegistry<uint32_t> comm_restart_cb_registry_;
+  std::function<std::string(uint32_t)> getPeerPubKey_;
```

Comment:
Please follow the coding standard, snake_case variables.

Reply:
fixed